tests/ui/traits/issue-77982.stderr

```diff
@@ -52,8 +52,32 @@ LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(0u32.into())).collect(
            - impl From<u8> for u32;
 help: try using a fully qualified path to specify the expected types
    |
-LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<T>>::into(0u32))).collect();
-   |                                                      +++++++++++++++++++++++    ~
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<u64>>::into(0u32))).collect();
+   |                                                      +++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<u128>>::into(0u32))).collect();
+   |                                                      ++++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<i64>>::into(0u32))).collect();
+   |                                                      +++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<i128>>::into(0u32))).collect();
+   |                                                      ++++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<f64>>::into(0u32))).collect();
+   |                                                      +++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<Ipv4Addr>>::into(0u32))).collect();
+   |                                                      ++++++++++++++++++++++++++++++    ~
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     let ips: Vec<_> = (0..100_000).map(|_| u32::from(<u32 as Into<AtomicU32>>::into(0u32))).collect();
+   |                                                      +++++++++++++++++++++++++++++++    ~
```

Comment:
This feels *unbelievably* verbose, and definitely a diagnostics regression. I do not think there's a reason to have all of these helps *and* the list of impls that satisfy `u32: From<_>` above too. IMO We should suggest writing `<u32 as Into</* type */>>::into` and then below write out the list of impls satisfying `u32: From<_>` for the user to pick from